src/birdie/internal/titles.gleam

```diff
@@ -402,7 +408,11 @@ fn try_fold_expression(
 
     glance.RecordUpdate(module: _, constructor: _, record:, fields:) -> {
       use acc <- result.try(try_fold_expression(record, acc, fun))
-      use acc, #(_field, expression) <- list.try_fold(over: fields, from: acc)
+      use acc, glance.RecordUpdateField(label: _field, item: expression) <- list.try_fold(
+        over: fields,
+        from: acc,
+      )
+      let assert Some(expression) = expression
```

Comment:
Why is this asserting? Is it safe to do so or would birdie start crashing on certain inputs

Reply:
The expression is `None` here when a shorthand is used so i just return `Ok(acc)` since it is the end of the expression